components/eam/src/physics/cam/physics_types.F90

```diff
@@ -1324,6 +1329,7 @@ subroutine physics_state_copy(state_in, state_out)
     do k = 1, pver
        do i = 1, ncol
           state_out%t(i,k)         = state_in%t(i,k) 
+          state_out%pv(i,k)        = state_in%t(i,k) 
```

Comment:
This looks like a typo, should be `state_in%pv`?

Reply:
It was; I found this a while back but must've missed the commit. Thanks for the catch!